app/frontend/helpers/i18n_helper.js

```diff
@@ -67,21 +68,15 @@ module.exports.getLocalizedDate = function(timestamp) {
   return new Date(Date.parse(timestamp)).toLocaleDateString(locale);
 }
 
-function toTitleCase(str) {
-  return str.slice(0, 1).toLocaleUpperCase() + str.slice(1);
-}
-
 module.exports.getLanguageName = function(code, includeNativeName = false, currentLocale = null) {
```

Comment:
How about adding some JSDoc comments for this function, since it seems a bit involved.